### DISCUSSION TITLE
Beta 3 Released

### DISCUSSION THREAD

**Discussion by lemonpole** [playliga/prototype#264] [Announcements]
- [View the full changelog](https://github.com/lemonpole/liga-public/releases/tag/v3.0.0-beta.3)

## Post Match Modal

![postgame](https://github.com/user-attachments/assets/2b7535f7-8748-4761-a7e4-e74ac218365f)

Beta 3 features an all new post-match modal, inspired by popular Counter-Strike Esports platforms like [hltv.org](https://www.hltv.org/) or [dust2.us](https://www.dust2.us/).

The app's `scorebot` module has been enhanced to track more than just scores. It now captures every kill, death, defusal, and bomb explosion during a match, displaying them all in the post-match modal for a more immersive experience.

## Settings Screen Keyboard Shortcut

A hopefully useful QoL addition. The settings screen now can be opened by typing `Win+,` on Windows or `Cmd+,` on MacOS.

## Match Historials

![historial](https://github.com/user-attachments/assets/77273850-5ef7-4d68-af64-0d83b1fc7c3d)

The match preview has been updated to also include each team's last five match results.

This new addition is designed to enhance immersion while also giving you valuable insights into your next opponent's current form.

**Comment by Bhrcs** [comment]
I know the game is just in beta but how far in the game are you able to go as of beta 3 cause I get to September 2026 and my game doesn't allow me to play my next match to continue

**Reply by lemonpole** [reply]
Hello, thanks for giving the game a try and sorry it stopped working for you!

You should be able to advance forever, technically. I've not implemented a limit, yet.

Do you have a match scheduled and when you click play it doesn't load up for you? If so, double check in the settings screen that your steam path is correct.

Other than that, I'd say please report the bug from the app's help screen as that will provide the app's logs file for me to see what errors may be happening under the hood.
